tutorials/observing_examples.ipynb

```diff
@@ -0,0 +1,441 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# <center>Computing the count rate of observing <a href=\"https://en.wikipedia.org/wiki/HAT-P-11\">HAT-P-11</a> with the APO 3.5m telescope using <tt>synphot</tt></center>"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 128,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%matplotlib inline\n",
+    "import matplotlib.pyplot as plt\n",
+    "import numpy as np\n",
+    "\n",
+    "import astropy.units as u\n",
+    "from astropy.io import fits\n",
+    "from astropy.modeling.blackbody import blackbody_lambda\n",
+    "\n",
+    "import synphot.units as units\n",
+    "from synphot.models import BlackBody1D, BlackBodyNorm1D, Empirical1D\n",
```

Comment:
Let's remove any imports we don't need anymore, like `BlackBodyNorm1D`, `blackbody_lambda`